core-api/src/worker.rs

```diff
@@ -95,6 +95,12 @@ pub struct WorkerConfig {
     /// todo: link to feature docs
     #[builder(default = "false")]
     pub use_worker_versioning: bool,
+
+    /// If set true, shutdown will not finish until all pending evictions have been issued and
+    /// replied to. If not set (the default) shutdown will be considered complete when the only
+    /// remaining work is pending evictions.
```

Comment:
Any reason not to set it to true by default? I thought the only reason was core tests.

Reply:
I'd also document why this flag is here

Reply:
Yeah, I can make it the default.